wfe2/wfe_test.go

```diff
@@ -293,6 +293,28 @@ var testKeyPolicy = goodkey.KeyPolicy{
 	AllowECDSANISTP384: true,
 }
 
+// inMemNonceService implements noncepb.NonceServiceClient for tests.
```

Comment:
This adapter could live in //test/inmem/nonce/nonce.go, just so all inmem adapters live in the same place. This one is pretty small, so it's maybe not necessary, but consistency would be nice.